### ISSUE TITLE
Accessibility errors found in your template

### ISSUE THREAD

**Issue by wapyce** [lcfu1/lcfu1#1]
The user @carlsonsantana validate your site template "https://lcfu1.github.io/lcfu1/" and found these accessibility errors:
* Page https://lcfu1.github.io/lcfu1/:
  * Issue *0232619e-57a6-4cf3-9b49-3c7eed13c539*:
    * **Type**: Error
    * **Code**: WCAG2AA.Principle1.Guideline1_4.1_4_3.G18.Fail
    * **Message**: This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 4.5:1, but text in this element has a contrast ratio of 4.15:1. Recommendation: change text colour to #2373db.
    * **Selector**: `html > body > footer > div > div:nth-child(3) > p > a`
    * **Context**: `<a href="https://github.com/lcfu1/lcfu1">lcfu1</a>`
  * Issue *3c6dbb25-b75a-47c2-bd8b-948fccd8e0ab*:
    * **Type**: Error
    * **Code**: WCAG2AA.Principle1.Guideline1_3.1_3_1.H49.AlignAttr
    * **Message**: Align attributes .
    * **Selector**: `html > body > footer > div > div:nth-child(3)`
    * **Context**: `<div align="center">
        <p>
            Copyri...</div>`
  * Issue *c90efbab-86c2-4f83-8fcc-d0ae715c62d4*:
    * **Type**: Error
    * **Code**: WCAG2AA.Principle1.Guideline1_4.1_4_3.G18.Fail
    * **Message**: This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 4.5:1, but text in this element has a contrast ratio of 3.78:1. Recommendation: change text colour to #757575.
    * **Selector**: `html > body > footer > div > div:nth-child(2) > div:nth-child(3) > p`
    * **Context**: `<p>描述你的网站</p>`
  * Issue *74296805-f823-4373-9477-a9f01a3e1fab*:
    * **Type**: Error
    * **Code**: WCAG2AA.Principle1.Guideline1_4.1_4_3.G18.Fail
    * **Message**: This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 4.5:1, but text in this element has a contrast ratio of 4.15:1. Recommendation: change text colour to #2373db.
    * **Selector**: `html > body > footer > div > div:nth-child(2) > div:nth-child(2) > ul > li:nth-child(2) > a > span`
    * **Context**: `<span>rss</span>`
  * Issue *f13bf0a7-36c7-4242-93e7-80759f9537e1*:
    * **Type**: Error
    * **Code**: WCAG2AA.Principle1.Guideline1_4.1_4_3.G18.Fail
    * **Message**: This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 4.5:1, but text in this element has a contrast ratio of 4.15:1. Recommendation: change text colour to #2373db.
    * **Selector**: `html > body > footer > div > div:nth-child(2) > div:nth-child(2) > ul > li:nth-child(1) > a > span`
    * **Context**: `<span class="username">lcfu1</span>`
  * Issue *aeb1e03d-0089-436b-aaa9-be08027a2fd2*:
    * **Type**: Error
    * **Code**: WCAG2AA.Principle1.Guideline1_4.1_4_3.G18.Fail
    * **Message**: This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 4.5:1, but text in this element has a contrast ratio of 4.15:1. Recommendation: change text colour to #2373db.
    * **Selector**: `html > body > footer > div > div:nth-child(2) > div:nth-child(1) > ul > li:nth-child(2) > a`
    * **Context**: `<a class="u-email" href="mailto:your-email@domain.com">your-email@domain.com</a>`
  * Issue *5f1e899d-c1ec-41ca-9909-e650d17edd9f*:
    * **Type**: Error
    * **Code**: WCAG2AA.Principle1.Guideline1_4.1_4_3.G18.Fail
    * **Message**: This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 4.5:1, but text in this element has a contrast ratio of 3.78:1. Recommendation: change text colour to #757575.
    * **Selector**: `html > body > footer > div > div:nth-child(2) > div:nth-child(1) > ul > li:nth-child(1)`
    * **Context**: `<li class="p-name">
            
              Use...</li>`
  * Issue *641c1dfe-89da-4f88-9e66-bc2cdd00d5ac*:
    * **Type**: Error
    * **Code**: WCAG2AA.Principle1.Guideline1_4.1_4_3.G18.Fail
    * **Message**: This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 4.5:1, but text in this element has a contrast ratio of 4.15:1. Recommendation: change text colour to #2373db.
    * **Selector**: `html > body > footer > div > h2 > a:nth-child(2)`
    * **Context**: `<a style="float: right; padding-right: 30px;" href="javascript:history.back(-1)">返回</a>`
  * Issue *2e3e90b7-4ad3-4514-8942-7509e47dd12b*:
    * **Type**: Error
    * **Code**: WCAG2AA.Principle1.Guideline1_4.1_4_3.G18.Fail
    * **Message**: This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 4.5:1, but text in this element has a contrast ratio of 4.15:1. Recommendation: change text colour to #2373db.
    * **Selector**: `html > body > footer > div > h2 > a:nth-child(1)`
    * **Context**: `<a style="float: right;" href="javascript:scroll(0,0)">顶部</a>`

You can check these accessibility errors using [pa11y](https://github.com/pa11y/pa11y).
You can view the [full validation results in our website](https://wapyce.herokuapp.com/validation/validations/333deeec-79c3-422b-b442-74a58e50739e/).
